azure/meta.go

```diff
@@ -0,0 +1,131 @@
+package azure
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/hairyhenderson/gomplate/v4/env"
+)
+
+// DefaultEndpoint is the IMDS endpoint at Azure
+// https://learn.microsoft.com/en-us/azure/virtual-machines/instance-metadata-service
+var DefaultEndpoint = "http://169.254.169.254"
```

Comment:
I know this is copied from `gcp/meta.go`, but it doesn't need to be exported (in either case):

```suggestion
// defaultEndpoint is the IMDS endpoint at Azure
// https://learn.microsoft.com/en-us/azure/virtual-machines/instance-metadata-service
var defaultEndpoint = "http://169.254.169.254"
```